README.md

```diff
@@ -32,8 +32,33 @@ You'll also need to clone a new `.env` file from the `.env.template` to store lo
 $ cp .env.template .env  # (first time only)
 ```
 
+As we will be using the Trello API, add the requests library to your list of poetry dependencies in pyproject.toml by running:
+
+'''bash 
+ $ poetry add requests
+'''
+poetry
 The `.env` file is used by flask to set environment variables when running `flask run`. This enables things like development mode (which also enables features like hot reloading when you make a file change). There's also a [SECRET_KEY](https://flask.palletsprojects.com/en/1.1.x/config/#SECRET_KEY) variable which is used to encrypt the flask session cookie.
 
+## Setting up Trello and configuring API token
+
+Trello’s API to fetch and save to-do tasks. In order to use their API, you need to first create an account (https://trello.com/signup), then generate an API key and token by following the instructions (https://trello.com/app-key).
+
+In the .env file, set the Trello API key and token as environment variables.
+Ensure that the .env file is added to the .gitignore to prevent credentials from being made public.
```

Comment:
A nice thing to add here is to add the keys for this to your `.env.template`, so we don't share the sensitive values, but we do give some clear advice for what needs to be set:
```
TRELLO_BOARD_ID=
TRELLO_KEY=
TRELLO_TOKEN=
```

Reply:
Added to the .env.template